parsers/events.js

```diff
@@ -62,11 +64,12 @@ class EventList {
    *
    * @param {Map<String, Object[]>} phrase The phrase to pick events from.
    * @param {Object} longestEvent The longest (by content length) and first voice event from phrase.
+   * @param {number} startIndex The index for the start of this event.
    * @return {String[]} voiceAdded Voice names added to event list.
-   * @return {Object[]} eventList List of the first of each voice from phrase that fit within bounds of longest event.
+   * @return {Object[]} line List of the first of each voice from phrase that fit within bounds of longest event.
```

Comment:
I used to call this a segment. It's essentially an indivisible (doesn't internally wrap) thing, like:

```
Am
Brown
```

Is that what you mean by this?

Reply:
I meant more of a line of a phrase, or a list of segments. For example,

```
c1:                    Am     G  F          G      Esus4  E
v1: All the leaves are brown        and the sky is gray

c1: F               C     E  Am       F        Esus4  E
v1: I've been for a walk         on a winter's day

c1:                 Am    G  F       G      Esus4  E
v1: I'd be safe and warm        if I was in L.A.

c1:            Am        G  F     G               Esus4  E
v1: California dreamin'        on such a winter's day
```

a line would be 

```

                   Am     G  F          G      Esus4  E
All the leaves are brown        and the sky is gray
```